### PR TITLE
Documentation fixup arm9/image.h, nds/tmers.h

### PR DESCRIPTION
**Pull request by awiebe**
- Clarification of arm9/image.h preamble
- Added missing spaces between words
- Spelling corrections
- SI Unit corrections

### CODE REVIEW

File: include/nds/arm9/image.h [hunk 1]
```diff
@@ -26,9 +26,9 @@
 	\brief An image abstraction for working with image data.
 
 <div class="fileHeader">
-Image data pointers must be allocated using malloc as the conversion
-rutiens will free the pointers and allocate new data.  As such any loader
-implemented utilizing this structure must use malloc() to allocate the image
+Image data buffers must be allocated using malloc(), rather than pointing to stack data, as the conversion
+routiens will free() the argument's image buffer and allocate a new block for the replacement data.
```
Comment (mtheall) [inline include/nds/arm9/image.h]: "routines"

File: include/nds/arm9/image.h [hunk 2]
```diff
@@ -26,9 +26,9 @@
 	\brief An image abstraction for working with image data.
 
 <div class="fileHeader">
-Image data pointers must be allocated using malloc as the conversion
-rutiens will free the pointers and allocate new data.  As such any loader
-implemented utilizing this structure must use malloc() to allocate the image
+Image data buffers must be allocated using malloc(), rather than pointing to stack data, as the conversion
```
Comment (mtheall) [inline include/nds/arm9/image.h]: Remove comma after "malloc()"

File: include/nds/arm9/image.h [hunk 3]
```diff
@@ -26,9 +26,9 @@
...
+routiens will free() the argument's image buffer and allocate a new block for the replacement data.  
+As such any loader implemented utilizing this structure must use malloc() to allocate the image
```
Comment (mtheall) [inline include/nds/arm9/image.h]: Add a comma after "As such"

File: include/nds/arm9/image.h [hunk 4]
```diff
@@ -26,9 +26,9 @@
...
 pointer data.
```
Comment (mtheall) [inline include/nds/arm9/image.h]: Replace "pointer data" with "buffer" to be consistent with terminology used earlier.

File: include/nds/arm9/image.h [hunk 5]
```diff
@@ -70,17 +70,17 @@ typedef struct sImage
 extern "C" {
 #endif
 
-/*! \brief Converts a 24 bit image to 16 bit
+/*! \brief Destructively converts a 24 bit image to 16 bit
```
Comment (mtheall) [inline include/nds/arm9/image.h]: Hyphenate "24-bit" and "16-bit"

File: include/nds/arm9/image.h [hunk 6]
```diff
@@ -70,17 +70,17 @@ typedef struct sImage
...
 	\param img a pointer to image to manipulate
 */
 void image24to16(sImage* img);
 
-/*! \brief Converts an 8 bit image to 16 bit setting the alpha bit
+/*! \brief Destructivley converts an 8 bit image to 16 bit setting the alpha bit
```
Comment (mtheall) [inline include/nds/arm9/image.h]: "Destructively"
Hyphenate "8-bit" and "16-bit"
Add a comma after "16-bit"

File: include/nds/arm9/image.h [hunk 7]
```diff
@@ -70,17 +70,17 @@ typedef struct sImage
...
 	\param img a pointer to image to manipulate
 */
 void image8to16(sImage* img);
 
-/*! \brief Converts an 8 bit image to 16 bit with alpha bit cleared for the supplied palette index
+/*! \brief Destructively converts an 8 bit image to 16 bit with alpha bit cleared for the supplied palette index
```
Comment (mtheall) [inline include/nds/arm9/image.h]: Hyphenate "8-bit" and "16-bit"

File: include/nds/timers.h [hunk 8]
```diff
@@ -73,7 +73,7 @@ extern "C" {
 /*!	\see TIMER_CR(n)
 	\see TIMER_FREQ(n)
 
-	%TIMER_DATA(n) when set will latch that value into the counter.  Everytime the
+	%TIMER_DATA(n) when set will latch that value into the counter.  Every time the
 	counter rolls over %TIMER_DATA(0) will return to the latched value.  This allows
```
Comment (mtheall) [inline include/nds/timers.h]: Add comma after "over"

File: include/nds/timers.h [hunk 9]
```diff
@@ -202,9 +202,9 @@ typedef enum {
 
 
 /*!	\brief start a hardware timer.
-	Callback is tied directly to interupt table and called directly resulting in less latency than the attached timer.
+	Callback is tied directly to interrupt table and called directly resulting in less latency than the attached timer.
```
Comment (mtheall) [inline include/nds/timers.h]: Add comma after "directly"

File: include/nds/timers.h [hunk 10]
```diff
@@ -202,9 +202,9 @@ typedef enum {
...
 	\param timer 	The hardware timer to use (0 - 3).
-	\param divider 	The timer channel clock divider (clock will tick at 33.513982 Mhz / divider)
+	\param divider 	The timer channel clock divider (clock will tick at 33.513982 MHz / divider)
 	\param ticks	The number of ticks which must elapse before the timer overflows
 	\param callback The callback to be called when the timer expires (if null no irq will be generated by the timer)
```
Comment (mtheall) [inline include/nds/timers.h]: Add comma after "if null"

**Comment by AntonioND** [comment]
Remove all the unnecessary commits.

**Comment by fincs** [comment]
Don't worry about that, we can very easily do squash and merge with GitHub's PR merge button.

**Pull request merged**